Engine/Engine.cs

```diff
@@ -164,7 +164,7 @@ public void Run(AlgorithmNodePacket job, AlgorithmManager manager, string assemb
                         initializeComplete = false;
                         //Get all the error messages: internal in algorithm and external in setup handler.
                         var errorMessage = String.Join(",", algorithm.ErrorMessages);
-                        errorMessage += String.Join(",", _algorithmHandlers.Setup.Errors);
+                        errorMessage += String.Join(",", _algorithmHandlers.Setup.Errors.Select(e => e.Message));
```

Comment:
If it only pass the `Message`, the stack trace will not be shown.

Reply:
Yup, this was just a mechanical refactoring and the original implementation didn't show the stack trace (since it wasn't communicated). If we want to modify it to change the stack we can do so in a separate PR.